spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -1220,6 +1225,10 @@ public void afterPropertiesSet() {
 		catch (IllegalStateException e) {
 			this.logger.debug("Could not enable micrometer timers", e);
 		}
+		if (this.isAsyncReplies() && !AcknowledgeMode.MANUAL.equals(this.acknowledgeMode)) {
+			this.acknowledgeMode = AcknowledgeMode.MANUAL;
+			logger.warn("Async replies require AcknowledgeMode.MANUAL, forced");
```

Comment:
We don't do `this.` for method and I think this message should go to `info()`.
Doesn't look like it cause some problem for a target solution having it be reconfigured properly.
I would say it even easier for me just have one option to set - `asyncReplies` and have everything else reconfigured properly.